src/tools/update/main.c

```diff
@@ -113,6 +126,9 @@ static char *convert_cygwin_path(const char *path) {
 		}
 	}
 
+	*out++ = '"';
+	*out++ = 0;
+
 	return cygpath;
```

Comment:
Ah, we have double quotes here, now. I added the call to `va` down below, which I think is a little cleaner (fewer lines).